fix typo in chapter_deep-learning-computation/parameters(_origin).md

YaoPengCN opened the pull request:
This pull request is to fix typo in "chapter_deep-learning-computation/parameters.md" and "chapter_deep-learning-computation/parameters_origin.md"

goldmermaid commented:
Hi @YaoPengCN ! Thanks for raising the issues and the solutions. I updated the English version a bit in this [commit](https://github.com/d2l-ai/d2l-zh/pull/988/commits/fa3d4fa23e30625f595814d2d46e1be15780b530). If you are okay with my changes, could you help adopt to the Chinese version as well! Thanks in advance!

YaoPengCN commented:
> Hi @YaoPengCN ! Thanks for raising the issues and the solutions. I updated the English version a bit in this [commit](https://github.com/d2l-ai/d2l-zh/pull/988/commits/fa3d4fa23e30625f595814d2d46e1be15780b530). If you are okay with my changes, could you help adopt to the Chinese version as well! Thanks in advance!

Okay, I have updated the Chinese version in "parameters.md".

goldmermaid commented:
Thanks @YaoPengCN !
